README.md

```diff
@@ -1,25 +1,108 @@
 # Quarkus Backstage
 
-[![Version](https://img.shields.io/maven-central/v/io.quarkiverse.backstage/quarkus-backstage?logo=apache-maven&style=flat-square)](https://search.maven.org/artifact/io.quarkiverse.backstage/quarkus-backstage)
+[![Version](https://img.shields.io/maven-central/v/io.quarkiverse.backstage/quarkus-backstage?logo=apache-maven&style=flat-square)](https://central.sonatype.com/artifact/io.quarkiverse.backstage/quarkus-backstage-parent)
 
-## Welcome to Quarkiverse!
+Generate Backstage Catalog Information as part of the Quarkus build or the Quarkus CLI.
```

Comment:
I propose that we rephrase the sentence to mention that we mainly create the Component YAML

"Generate the Backstage Catalog Information as the [component](https://backstage.io/docs/features/software-catalog/descriptor-format/) as part of the Quarkus build or the Quarkus CLI.

Reply:
I don't agree with the proposal. It feels way too verbose.
We don't need to be more specific than that. Moreover, the catalog-info.yaml may contain other entities, too.